pkg/api/interceptor.go

```diff
@@ -45,9 +45,22 @@ func (wa *WalletAuthorizer) Unary() grpc.UnaryServerInterceptor {
 		info *grpc.UnaryServerInfo,
 		handler grpc.UnaryHandler,
 	) (interface{}, error) {
+		if !wa.Ratelimits && !wa.requiresAuthorization(req) {
```

Comment:
@richardhuaaa recently added an escape hatch for V3 messages, because they don't have auth tokens. But now that we are rate limiting by IP, we can include them again.

Reply:
Yup, we can still keep the flag to avoid authorizing V3 publishes and then we can let them through the same limit flow. They won't be able to get priority limits, but that may be fine in the short term.

Reply:
Considering there are maybe 5 addresses in the entirety of the V3 address space right now, I think we're a long way from needing to worry about priority limits.